templatizing strings for translation

brainwane opened the pull request:
Per [bug 30](https://fedorahosted.org/hyperkitty/ticket/30), I have systematically found all, or nearly all, user-visible occurrences of hard-coded English strings in the `hyperkitty/templates/hyperkitty/` directory and prepared them for translation/localisation/i18n as follows:

` {% trans 'English message' %}`

In some cases I have not been sure how to handle escaping, quoting, interaction with other variables, or pluralization; I'm happy to update this pull request in response to code review. Thanks.

abompard commented:
This looks nice, thanks for the work! However, the `trans` tag requires the `{% load i18n %}` tag at the top of the template file, and it is not always present. Could you add it in the missing files?

brainwane commented:
Thanks for the heads-up. Doing that now.

brainwane commented:
abompard, I've added the `{% load i18n %}` command to the files in `templates/` where it wasn't present -- and in `base.html` I've moved it up from line 23 to line 2.

abompard commented:
Hey Sumana,
Sorry about the delay, but in the meantime I've change a few things on the templates, could you rebase your changes on the current HEAD? Thanks!

brainwane commented:
Here you go - rebased!
